.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,118 @@
+# ========================================================================================================
+# Customizing Gource
+# specific screen resolution
+gource -1280x1024 --camera-mode track --hide progress --output-ppm-stream ./out.ppm
+--background-colour 000000 --font-colour 336699 --highlight-users --highlight-colour ff0000
+--auto-skip-seconds 1 --seconds-per-day 0.051 --title "hmm..." --user-scale 1
+--bloom-multiplier 0.5 --bloom-intensity 0.5 --key
+--file-extensions repo.log
+
+# (?) WHY fullscreen is too slow
+gource -fullscreen --camera-mode track --hide progress --output-ppm-stream ./out.ppm
+--background-colour 000000 --font-colour 336699 --highlight-users --highlight-colour ff0000
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)